src/Moryx.AbstractionLayer.Products.Endpoints/ProductManagementController.cs

```diff
@@ -161,10 +161,17 @@ public ActionResult<ProductModel> GetTypeById(long id)
         {
             if (id == 0)
                 return BadRequest($"Id was 0");
-            var productType = _productManagement.LoadType(id);
-            if (productType == null)
+            try
```

Comment:
This solves the problem, but the function should not throw an exception 🤔 in this line we see that null will be returned if the product could not be found
https://github.com/PHOENIXCONTACT/MORYX-AbstractionLayer/blob/080eb5cc8a40838906b600003b2fe626604b3185/src/Moryx.Products.Management/Implementation/Storage/ProductStorage.cs#L462

Where is the exception raised? 
@slawor if its the intended behavior we should extend the xml comment of the facade/productmanager/productstorage that its possible to get an exception 🤔